datahub_sap_hana/column_lineage_schema.py

```diff
@@ -40,11 +41,17 @@ class ColumnField:
     dataset: Table
 
     @classmethod
-    def from_node(cls, node: Node, schema: str):
+    def from_node(cls, node: Node, default_schema: str):
         """Creates a ColumnField from a sqlglot node."""
+
+        schema = default_schema
+
+        if isinstance(node.source, expressions.Table):
+            schema = node.source.catalog or node.source.db  # type:ignore
```

Comment:
it might also be that both catalog and db are empty, so this should also include the default_schema
```suggestion
            schema = node.source.catalog or node.source.db or default_schema # type:ignore
```

is the type: ignore really needed? i thought the `isinstance` would be enough to make the linter happy.